apps/studio/pages/project/[ref]/database/backups/restore-to-new-project.tsx

```diff
@@ -202,6 +202,20 @@ const RestoreToNewProject = () => {
     )
   }
 
+  if (isFly) {
+    return (
+      <Admonition
+        type="default"
+        title="Restoring to new projects are not available for Fly hosted projects"
+      >
+        <Markdown
+          className="max-w-full [&>p]:!leading-normal"
+          content={`If you need to a Fly project, please reach out via [support](/support/new?ref=${project?.ref}).`}
+        />
```

Comment:
@jordienr i'm currently working on winding down fly postgres so you can remove this line.